tests/Feature/TourSerializerTest.php

```diff
@@ -21,12 +25,15 @@ public function test_serialize(): void
             $this->assertArrayHasKey('translations', $tour);
             $this->assertArrayHasKey('location', $tour);
             $this->assertArrayHasKey('latitude', $tour);
+            $this->assertIsString($tour['latitude']);
             $this->assertArrayHasKey('longitude', $tour);
+            $this->assertIsString($tour['longitude']);
             $this->assertArrayHasKey('floor', $tour);
             $this->assertArrayHasKey('image_url', $tour);
             $this->assertArrayHasKey('thumbnail_full_path', $tour);
             $this->assertArrayHasKey('large_image_full_path', $tour);
             $this->assertArrayHasKey('selector_number', $tour);
+            $this->assertIsString($tour['selector_number']);
```

Comment:
`selector_number`, `latitude` and `longitude` are strings and not numeric fields?

Reply:
Yup, that's how they're typed in the JSON:
![Screenshot 2023-09-29 at 10 41 06 AM](https://github.com/art-institute-of-chicago/aic-mobile-cms/assets/2098951/8ffc0a60-65b9-4419-9198-aaae9fe8000a)